scripts/fast-reboot-dump.py

```diff
@@ -275,10 +281,17 @@ def send_garp_nd(neighbor_entries, map_mac_ip_per_vlan):
     # send arp/ndp packets
     for vlan_name, dst_mac, dst_ip in neighbor_entries:
         src_if = map_mac_ip_per_vlan[vlan_name][dst_mac]
+        if subtype.lower() != 'dualtor':
```

Comment:
This is technically ok, but contains maybe safer?

Reply:
Sure, addressed this now.